docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -42,100 +42,81 @@ Note that the cloud provider example used in this document has a cluster-level a
+- In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1 x 100GB.
 - Use the **Enable YSQL** field to enable the YSQL API endpoint to run Postgres-compatible workloads. This setting is enabled by default.
-
 - Use the **Enable YEDIS** field to enable the YEDIS API endpoint to run REDIS-compatible workloads. This setting is disabled by default.
-
-- Use the **Enable Node-to-Node TLS** field to enable encryption-in-transit for communication between the database servers. This setting is enabled by default in YugabyteDB Anywhere version 2.7.1.1 and later.
-
-- Use the **Enable Client-to-Node TLS** field to enable encryption-in-transit for communication between clients and the database servers. This setting is enabled by default in YugabyteDB Anywhere version 2.7.1.1 and later.
-
+- Use the **Enable Node-to-Node TLS** field to enable encryption-in-transit for communication between the database servers. This setting is enabled by default.
+- Use the **Enable Client-to-Node TLS** field to enable encryption-in-transit for communication between clients and the database servers. This setting is enabled by default.
 - Use the **Enable Encryption at Rest** field to enable encryption for data stored on the tablet servers. This setting is disabled by default.
-
-  ![img](/images/yb-platform/kubernetes-config6.png)
+- Enter the appropriate information in the remaining fields.
 
 Complete the **Advanced** section as follows:
 
 - In the **DB Version** field, specify the YugabyteDB version. The default is either the same as the YugabyteDB Anywhere version or the latest YugabyteDB version available for YugabyteDB Anywhere.
 
-- Use the **Enable IPV6** to enable the use of IPV6 networking for connections between the database servers. This setting is enabled by default.
+- Enter the appropriate information in the remaining fields.
 
-- Use the **Enable Public Network Access** field to assign a load balancer or nodePort for connecting to the database endpoints over the internet. This setting is enabled by default.
-
-  ![img](/images/yb-platform/kubernetes-config7.png)
 
 Complete the **G-Flags** section as follows:
 
-- Use the **Master** fields to specify YugabyteDB Masters parameters, one parameter per field (see [YB Master Configuration Flags](/preview/reference/configuration/yb-master/#configuration-flags) for details).
+- Click **Add Flags > Add to Master** to specify YugabyteDB Masters parameters, one parameter per field.
 
-- Use the **T-Server** fields to specify the YugabyteDB T-Servers parameters, one parameter per field (see [YB T-Server Configuration Flags](/preview/reference/configuration/yb-tserver/#configuration-flags) for details).
+- Click **Add Flags > Add to T-Server** to specify the YugabyteDB T-Servers parameters, one parameter per field.
 
-  ![img](/images/yb-platform/kubernetes-config8.png)
+  For details, see the following:
```

Comment:
Need a line break here? Runs in in Preview

Reply:
fixed